widgetastic_manageiq.py

```diff
@@ -1205,15 +1206,15 @@ class ItemsToolBarViewSelector(View):
         view_selector.selected
     """
     ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
-    grid_button = Button(title='Grid View')
-    tile_button = Button(title='Tile View')
-    list_button = Button(title='List View')
+    btn_class = VersionPick({
```

Comment:
OPTIONAL: wouldn't it be better to make one class ViewChangeButton which supports both 5.6 and 5.7 ?

abra kadabra summoning @mfalesni. What's your opinion ?